libs/core/models/anime.ts

```diff
@@ -0,0 +1,42 @@
+import { Immerable, OmitImmerable } from './immerable';
+
+/** Anime model. */
+export class Anime extends Immerable {
+
+	/** Anime id. */
+	public readonly id: number;
+
+	/** Anime preview image URL. */
+	public readonly imageUrl: string;
+
+	/** Anime English title. */
+	public readonly titleEnglish: string;
+
+	/** Anime Japanese title. */
+	public readonly titleJapanese: string;
+
+	/**
+	 * Anime start airing date.
+	 * @example '2024-07-22T02:40:54.873Z'.
+	 */
+	public readonly startDate: string | null;
```

Comment:
```suggestion
	public readonly startDate: Date | null;
```
It's much easier to work with a date type than with a string